Veracode Example.postman_collection.json

```diff
@@ -1,9 +1,11 @@
 {
 	"info": {
-		"_postman_id": "c743a345-73eb-4eca-b4dd-46ca1ee972f5",
+		"_postman_id": "5063f89b-8127-49b9-a6e6-fd83a4ed89e5",
```

Comment:
very interesting, wonder what information we're leaking there

Reply:
If you want to go down the rabbit hold: https://github.com/postmanlabs/postman-app-support/issues/2906
TL;DR it's a meaningless GUID that gets assigned upon import that shouldn't be getting exported with newer versions but it's buggy.